<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estudio sue침os</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbx1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTemperatura">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura y flujo</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClustering">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>690</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clustering &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbConsumo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Consumo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableDistancias">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wgDendograma" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matriz de distancias entre episodios de sue침o</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comparaci칩n entre episodios de sue침o</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>611</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="plotLayout"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
